plugins/lsp_java.lua

```diff
@@ -0,0 +1,45 @@
+-- mod-version:3 -- lite-xl 2.1
+
+local core = require "core"
+local common = require "core.common"
+local config = require "core.config"
+local lsp = require "plugins.lsp"
+
+local installed_path_plugin = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_java"
+local ok, jdk_info = pcall(require, "libraries.jdk")
+
+local platform
+if PLATFORM == "Windows" then
+  platform = "win"
+elseif PLATFORM == "Mac OS X" then
+  platform = "mac"
+else
+  platform = "linux"
+end
+
+local java_home = jdk_info.path_lib .. PATHSEP .. jdk_info.version
```

Comment:
At this point, we could just make `jdk_info.path_lib` contain the version.

Reply:
Why should they not remain separate?

Reply:
Without the version it's not really useful.

Reply:
Ok